src/CheckoutSdk/Payments/Links/BillingInformation.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Checkout.Common;
+using System;
+
+namespace Checkout.Payments.Links
+{
+    public class BillingInformation : IEquatable<BillingInformation>
+    {
+        public Address Address { get; set; }
+
+        public Phone Phone { get; set; }
+
+        public bool Equals(BillingInformation other)
+        {
+            if (ReferenceEquals(null, other)) return false;
+            if (ReferenceEquals(this, other)) return true;
+            return Address == other.Address && Phone == other.Phone;
```

Comment:
Address = null; other.Address = null; Address.Equals(other.Address) => throw exception null reference.

In my branch "support-reconciliation" I upload an extension to avoid this Equals null problem. It's called EqualsNull() and is able to compare even if one of them is equals.